tool/preprocess-entity-dataset.js

```diff
@@ -0,0 +1,80 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of Genie
+//
+// Copyright 2019-2020 The Board of Trustees of the Leland Stanford Junior University
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author: Naoki Yamamura <yamamura@cs.stanford.edu>
+"use strict";
+
+const fs = require('fs');
+const util = require('util');
+const assert = require('assert');
+const I18N = require('../lib/i18n');
+
+class EntityPreprocessor {
+    constructor(options) {
+        this._locale = options.locale;
+        this._tokenizer = I18N.get(options.locale).getTokenizer();
+    }
+
+    async process(fins, fout) {
+        const values = [];
+        for (const fin of fins) {
+            let input = JSON.parse(await util.promisify(fs.readFile)(fin), {encoding: 'utf8'});
```

Comment:
all this large file preprocessing is probably better to be done with streams... but this has lower priority to fix

Reply:
After clean up this file is not required. Removed.